### PR TITLE
Ent - Bug fix RMP-297

### PR DESCRIPTION
**Pull request by gayan-orangehrm**
This includes a bug fix for

1. Archive confirmation modal close icon “x“ is not center-aligned
2. Replaced button with "x" with the proper oxd icon

### CODE REVIEW

File: components/src/core/components/Input/Select/select-input.scss [hunk 1]
```diff
@@ -229,4 +229,8 @@
       overflow: hidden;
     }
   }
-}
\ No newline at end of file
+}
+
+.double-line-item-selected {
```
Comment (lgtkaushalya) [inline components/src/core/components/Input/Select/select-input.scss:234]: Discuss with Mike, why this is needed.

File: components/src/core/components/Button/Button.vue [hunk 2]
```diff
@@ -93,6 +93,7 @@ export default defineComponent({
         'oxd-button': true,
         [`oxd-button--${this.size}`]: true,
         [`oxd-button--${this.displayType}`]: true,
+        'with-icon': this.iconName || this.iconRightName
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Button/Button.vue:96]: fixed

File: components/src/core/components/Input/Select/select-input.scss (hunk 1 above)
Comment (gayan-orangehrm) [reply components/src/core/components/Input/Select/select-input.scss:234]: removed. I'll fix the styles after discussing with Mike

File: components/src/core/components/Dialog/CloseButton.vue [hunk 3]
```diff
@@ -1,12 +1,22 @@
 <template>
-  <button class="oxd-dialog-close-button">×</button>
+  <div class="oxd-dialog-close-button-wrapper d-flex align-center justify-center">
+    <oxd-icon-button
+      class="oxd-dialog-close-button-icon"
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Dialog/CloseButton.vue:4]: reverted "oxd-dialog-close-button-wrapper" back to "oxd-dialog-close-button"

**Pull request merged**